Neos.Flow/Classes/Cli/CommandManager.php

```
@@ -270,9 +270,10 @@ public static function getCommandControllerMethodArguments(ObjectManagerInterfac
                 continue;
             }
             $controllerObjectName = $objectManager->getObjectNameByClassName($className);
+            assert($controllerObjectName !== false);
```

Maybe rather throw an exception here?

---

adjusted